style.css

```
@@ -294,4 +294,83 @@ body {
     background-color: #dfe1e6;
     margin: 0;
   }
+
+  .form_section {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-image: url("/pictures/contactformshapes-mobile@2x.png");
```

- [ ] **[OPTIONAL]:** Please consider using relative paths over absolute ones, they are often easier to handle! 🤓 🔧 

```suggestion
    background-image: url("pictures/contactformshapes-mobile@2x.png");
```

or

```suggestion
    background-image: url("./pictures/contactformshapes-mobile@2x.png");
```